src/app/elements/RequestButton/RequestButton.tsx

```diff
@@ -0,0 +1,41 @@
+import { Theme } from '@libs/theme';
+
+function RequestButton(props: {
+  onClick: (e: React.SyntheticEvent<HTMLButtonElement>) => void;
```

Comment:
```suggestion
  onClick: (e: React.SyntheticEvent<HTMLButtonElement>) => void;
```

e를 실제로 받는곳이 없는데요...?

Reply:
변경했습니다~ 이 부분은 회원가입에서만 사용되는 부분이라 @LuisKlopp 루이 같이 확인 한 번 부탁드려용~! 제가 전에 루이 브랜치를 잘못 pull 받아서 생성되었습니당. 주의 하겠습니다.